toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/tools/test_xnvme.py

```diff
@@ -118,6 +126,8 @@ def test_log(cijoe, device, be_opts, cli_args):
 
     if be_opts["be"] == "linux" and be_opts["admin"] in ["block"]:
         pytest.skip(reason="[admin=block] does not implement get_log")
+    if "ramdisk" in device["labels"]:
+        pytest.skip(reason="[be=ramdisk] does not implement error-log")
```

Comment:
Should be `get_log` (or maybe `get-log`) instead of error-log

Reply:
Changing to get-log. thanks!